Helper/Config.php

```diff
@@ -292,4 +292,29 @@ public function initSdkConfiguration(bool $isAuthInfoRequired = true) : void
         SdkConfigManager::setValue(sprintf('%s.authorization.auth_user', $sdkEnv), $this->getAuthenticatedUserName());
         SdkConfigManager::setValue(sprintf('%s.authorization.access_token', $sdkEnv), (string) $this->getAccessToken());
     }
+
+    /**
+     * Retrieve a list of stores that the user has access to
+     *
+     * @return $stores array a list of stores
+     */
+    public function getStores()
+    {
+
+        $om = $this->context->getObjectManager();
+        $storeManager = $om->get('Magento\Store\Model\StoreManagerInterface');
+        $storeManagerDataList = $storeManager->getStores($withDefault = false);
+        $store = array();
+
+        foreach ($storeManagerDataList as $key => $value) {
```

Comment:
if the `$key` isn't used here, the `foreach` can be written as `foreach ($storeManagerDataList as $storeDataManager)`

Reply:
I fixed all the things!